### DISCUSSION TITLE
Private embed

### DISCUSSION THREAD

**Discussion by Xinnerpeace** [Budibase/budibase#12541] [Ideas]
Budibase allows building custom signup/login system, which is great.

However, it's handicapped in iFrames because "Only application Screens with Public Access level can be viewed within an embedded Budibase app."
This brings limits to embeded Budibase app from displaying private data, unpublic screens show warnings as "Session not authenticated".

https://docs.budibase.com/docs/embedded-app
https://docs.appsmith.com/advanced-concepts/embed-appsmith-into-existing-application

**Comment by melohagan** [comment]
Hey @Xinnerpeace 

I'm interested in what you are trying to accomplish; could you outline your use-case in a little more detail please?
